FALSEHOODS.md

```diff
@@ -118,3 +118,8 @@ surprising how many false assumptions programmers continue to make about them.
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  **Phone numbers have only one prefix**
```

Comment:
The old system had regional prefixes.  The new system has seven digit phone numbers and no regional codes.  So during the transition, a Reykjavik phone number could be dialed with nnnnn and 55nnnnn inside the region; and 91-nnnnn and 55nnnnn from outside.